andela/activo-api
# #157505126 Set up linting, Wiki and Readme

#### What does this PR do?
- Set up **pylint** , readme file and project wiki
#### Description of Task to be completed?
- Install pylint to lint code in accordance with PEP8
- Add a readme file
- Add a project Wiki to the Github repo
#### How should this be manually tested?
- Clone the application
- Run `pip install -r requirements.txt` to install all dependencies
- Add a python file and observe the linter pointing out where there is a deviation from the team convention. 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
 - ch/157505126/code conventions + linter + readme
#### Screenshots (if appropriate)
-N/A
#### Questions:
N/A
